### DISCUSSION TITLE
Welcome to Basemaps Discussions!

### DISCUSSION THREAD

**Discussion by Fanchengyan** [Fanchengyan/Basemaps#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by askost1210** [comment]
Hello, first of all I'd like to thank you for this plugin. could you please pull request for the maps on this sites(?): https://isogd.gorodperm.ru/#/map?lat=58.03&lng=56.25&zoom=11
https://isogd.permkrai.ru/#/desktop

**Comment by Fanchengyan** [comment]
Hi @askost1210 ,

Sorry, I can't understand the text on this website. Does this site offer services like XYZ? If it does, you can click the 'Add' button at the bottom of the plugin panel to add it manually.
